### ISSUE TITLE
A blank page after title page fools page numbering

### ISSUE THREAD

**Issue by mdykierek** [michaelrsweet/htmldoc#371]
If a blank page after title is added (e.g. for printing), then it gets numbered with 'i' in the viewers (e.g. Okteta, Foxit, Firefox, Adobe Reader) and the last page of TOC gets numbered with 1.
It makes numbers "printed" on pages differ from displayed in the viewers.
If one wants to go to page with number 1 (printed and listed in TOC) needs to enter 2 in the so called "go to" field.

**Comment by mdykierek** [comment]
An example HTML:
```
<html>
<head>
    <title>Title</title>
</head>
<body>
<!-- NEW PAGE -->

<h1><a name="h1_1"></a>1</h1>
<h2><a name="h2_1"></a>1.1</h2>
<p>&nbsp;</p>
<!-- NEW PAGE -->

<h2><a name="h2_2"></a>1.2</h2>
<p>&nbsp;</p>
</body>
</html>
```

The command line: htmldoc -f text.pdf -t pdf14 --book test.htm

**Comment by michaelrsweet** [comment]
Hmm, will look at fixing this...

**Comment by michaelrsweet** [comment]
Hmm, strange, the page numbers are correct for the footers but not for the page annotations...

**Comment by michaelrsweet** [comment]
[master 374cde3] Fix PDF page numbers (Issue #371)
